app/models/skill.rb

```diff
@@ -1,4 +1,9 @@
 class Skill < ActiveRecord::Base
+  belongs_to :group
+
+  validates :name, presence: true
+  validates :featured, inclusion: [true, false]
```

Comment:
true and false are going to come through as strings, will that make a difference here?